examples/adder/tests/test_adder.py

```diff
@@ -45,3 +54,35 @@ async def adder_randomised_test(dut):
         ), "Randomised test failed with: {A} + {B} = {X}".format(
             A=dut.A.value, B=dut.B.value, X=dut.X.value
         )
+
+
+def test_adder_runner():
+
+    hdl_toplevel_lang = os.getenv("HDL_TOPLEVEL_LANG", "verilog")
+    sim = os.getenv("SIM", "icarus")
+
+    proj_path = Path(__file__).resolve().parent.parent
+    sys.path.append(str(proj_path / "model"))
+
+    verilog_sources = []
+    vhdl_sources = []
+
+    if hdl_toplevel_lang == "verilog":
+        verilog_sources = [proj_path / "hdl" / "adder.sv"]
+    else:
+        vhdl_sources = [proj_path / "hdl" / "adder.vhdl"]
+
+    sys.path.append(str(proj_path / "tests"))
```

Comment:
```suggestion
    # equivalent to setting the PYTHONPATH environment variable
    sys.path.append(str(proj_path / "tests"))
```
Maybe helpful for newbies.